package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@idkjay/personal",
+  "version": "1.0.5",
+  "description": "Don't leave your messages unread! Wrap them with the eyes emoji.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/idkjay/2020-04-17-personal-npm-package.git"
+  },
+  "keywords": [
+    "node",
+    "npm",
+    "eyes"
+  ],
+  "author": "Jordan Chu <jordanchu95@gmail.com> (http://jordanchu.xyz)",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/idkjay/2020-04-17-personal-npm-package/issues"
+  },
+  "homepage": "https://github.com/idkjay/2020-04-17-personal-npm-package#readme",
+  "dependencies": {
```

Comment:
As you mentioned, I don't think you need all these dependencies on your package. Maybe delete them, since it makes your package bigger (heavier).

Reply:
I was trying to make something out of it but I couldn't figure out how to reference the whole npm package on codepen. :(